configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5s.sh

```diff
@@ -0,0 +1,72 @@
+ #!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DECODE="decodebin" #decodebin|vaapidecodebin
+DEVICE="CPU" #GPU|CPU|MULTI:GPU,CPU
+PRE_PROCESS="" #""|pre-process-backend=vaapi-surface-sharing|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 
```

Comment:
please default to some suitable value, asking the end developer to input all of that thing is too much IMO